.babelrc

```diff
@@ -1,7 +1,22 @@
 {
   "presets": [
-    "react",
-    "es2015",
-    "stage-2"
+    "@babel/preset-react",
+    "@babel/preset-env"
+  ],
+  "plugins": [
+    "@babel/plugin-syntax-dynamic-import",
+    "@babel/plugin-syntax-import-meta",
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-json-strings",
+    [
```

Comment:
Don't think we need all these plugins, do we?

Reply:
@ryanashcraft, Nope! That was just to get it to match stage-2. Dropped everything except `plugin-proposal-class-properties`.